doc/input-output-reference/src/standard-output-reports/output-table-summaryreports.tex

```diff
@@ -73,11 +73,22 @@ \subsubsection{Predefined Annual Summary Reports}\label{predefined-annual-summar
 
 \begin{itemize}
 \item
-  Opaque which includes all opaque surfaces and includes the name of the construction, reflectance, U-Factor, gross area, azimuth, tilt, cardinal direction.
+  Opaque exterior which includes all exterior opaque surfaces and includes the name of the construction, zone, reflectance, U-Factors, areas, azimuth, tilt, cardinal direction.
 \item
-  Exterior Fenestration which includes all non-opaque exterior surfaces and includes the name of the construction, areas (glass, frame, divider, single opening, multiplied openings), glass U-Factor, glass SHGC (the solar heat gain coefficient based on summer conditions), glass visible transmittance, NFRC Product Type, assembly U-Factor, assembly SHGC, assembly visible transmittance, conductance (frame, divider), indication of shade control, the name of the parent surface, azimuth, tilt, cardinal direction. 
+  Opaque exterior which includes all exterior opaque surfaces and includes the name of the construction, zone, adjacent surface, reflectance, U-Factors, areas, azimuth, tilt, cardinal direction.
+\item
+  Exterior Fenestration which includes all non-opaque exterior surfaces and includes the name of the construction, frame and divider name, areas (glass, frame, divider, single opening, multiplied openings), glass U-Factor, glass SHGC (the solar heat gain coefficient based on summer conditions), glass visible transmittance, NFRC Product Type, assembly U-Factor, assembly SHGC, assembly visible transmittance, conductance (frame, divider), indication of shade control, the name of the parent surface, azimuth, tilt, cardinal direction. 
 \item
   Exterior Fenestration Shaded State has values shown for exterior fenestration that have a shaded state by using \hyperref[windowpropertyshadingcontrol]{WindowShadingControl} the values of the u-factor, SHGC, and visible transmittance are shown for both the glass only and the full assembly when in the shaded state.
+\item
+  Interior Fenestration which includes the construction, areas, glass u-factor, SHGC, and visible transmittance, and the parent surface name.
+\item
+  Exterior Door which includes the construction, u-factors, area, and the parent surface name.
```

Comment:
I don't think there is enough information to figure out swinging/nonswining for doors so it would have to be something that the user provides as a compliance parameter. I didn't actually touch this table on exterior doors I just noticed that it wasn't documented but from my code review it doesn't seem like GlassDoors are being reported here I think they end up being reported as exterior fenestration.